### ISSUE TITLE
Dart Solo Jiango Crash

### ISSUE THREAD

**Issue by zack37** [Legend-of-Dragoon-Modding/Severed-Chains#1541]
While playing with Dart as only party member, I got a crash when Jiango did his jumping all party damage skill even though the spreadsheet for Party Testing says Jiango should be doable as long as Dart is in the party
 
```
22:01:01.771 [Hardware legend.game.combat.SEffe:516] INFO : Deallocating effect manager 13
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:184] ERROR: Script 17 crashed!
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:185] ERROR: File 5204/1[addr 0x37ac]
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:186] ERROR: Parameters:
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:187] ERROR:   Op param: 0x6
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:189] ERROR:   1: script[0xdef] 0x40
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:189] ERROR:   2: null
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:189] ERROR:   3: null
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:192] ERROR: Storage:
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   1: 0x11
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   2: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   3: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   4: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   5: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   6: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   7: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   8: 0x90000
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   9: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   10: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   11: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   12: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   13: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   14: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   15: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   16: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   17: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   18: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   19: 0x13
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   20: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   21: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   22: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   23: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   24: 0x0
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   25: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   26: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   27: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   28: 0xb
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   29: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   30: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   31: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   32: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   33: 0xffffffff
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:197] ERROR: Call stack:
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   1: 0
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   2: 0
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   3: 0
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   4: 0
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   5: 0
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   6: 0
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   7: 0
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   8: 0
22:01:01.772 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   9: 0
22:01:01.772 [Hardware legend.core.RenderEngine:1123] ERROR: Shutting down due to exception:
java.lang.RuntimeException: An error occurred while ticking script 17
	at legend.game.scripting.ScriptManager.executeScriptFrame(ScriptManager.java:206)
	at legend.game.scripting.ScriptManager.tick(ScriptManager.java:42)
	at legend.game.Scus94491BpeSegment.lambda$gameLoop$45(Scus94491BpeSegment.java:410)
	at legend.core.RenderEngine.lambda$init$17(RenderEngine.java:494)
	at legend.core.opengl.Window$Events.onDraw(Window.java:697)
	at legend.core.opengl.Window.tickFrame(Window.java:337)
	at legend.core.opengl.Action.run(Action.java:46)
	at legend.core.opengl.Action.tick(Action.java:32)
	at legend.core.opengl.Window.run(Window.java:314)
	at legend.core.RenderEngine.run(RenderEngine.java:1121)
	at legend.core.GameEngine.start(GameEngine.java:254)
	at legend.game.Main.main(Main.java:23)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 72
	at legend.game.scripting.ScriptState.executeFrame(ScriptState.java:378)
	at legend.game.scripting.ScriptManager.executeScriptFrame(ScriptManager.java:180)
	... 11 more
```

**Comment by avionanx** [comment]
Party testing is slightly under-documented, use [this](https://docs.google.com/spreadsheets/d/14gH3SDnXKe929-nQTsmmjhvjOiIQbsgYT_hYKEkfbfg/) instead. None of these fixes have been merged to RB3 yet

**Comment by Chocolate06** [comment]
Damn, I suck.

**Comment by avionanx** [comment]
Closed by #1558 and #1572
